### DISCUSSION TITLE
docs/query/api/

### DISCUSSION THREAD

**Discussion by giscus** [cloudfly/victoriametrics-docs#2] [Announcements]
# docs/query/api/

VictoriaMetrics 的数据查询 API 接口文档说明和样例，方便进行日常参考。

https://victoriametrics.com.cn/docs/query/api/

**Comment by nolesse** [comment]
/api/v1/query_range 查询响应的第一个点为什么不是从start_time开始的呢，prometheus是这样的

**Reply by leiru0806** [reply]
hello,请问这个问题有解决吗？

**Reply by cloudfly** [reply]
对 timestamp 取整的行为，这里有一些说明介绍：https://www.victoriametrics.com.cn/docs/query/#instant-query
